docs/concepts/demisto-sdk.md

```diff
@@ -0,0 +1,388 @@
+    ```
+   This command will validate all committed and staged files and will print out some additional information which is hidden by default: 
+    - Which files were ignored.
+    - Error codes that were ignored.
+    - A report on which files where identified and in what status via `git`.
+   This additional information might be helpful in debugging SDK commands or trying to figure out why a file was not validate of some reason.
+
+#### Validate Error Structure and Ignoring Errors
+Each one of our validation errors follows a basic structure to ease it's readability:
+```buildoutcfg
+File_Path: [Error_Code] - Error Message
+```
+
+For example:
+```buildoutcfg
+Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.yml: [BA100] - The version for our files should always be -1, please update the file.
+```
+
+At the end of the validation a summary would appear with all the file paths and error codes that were encountered.
+
+The error codes serve two main functions:
+1. The first two letters are used to identify the type of the error encountered for example: `DO` is a Docker related error, `BC` is backwards compatibility related and `ST` is a yml/json structure error.
+    A full list of abbreviations can be found in the full validate documentation linked below.
+2. The error code is used to ignore errors. To ignore an error go to the file `.pack-ignore` file and input the following structure:
+   ```buildoutcfg
+    [file:file_name]
+    ignore=<error_code_to_ignore>
+    ```
+   For example:
+   ```buildoutcfg
+    [file:CortexXDRIR.yml]
+    ignore=IN126,IN135
+    ```
+   * Note: **Not all error codes are ignorable! Also, it is always preferable to fix the error rather than trying to ignore it!** 
+   Please consult with a Cortex XSOAR team member before ignoring an error.  
+
+For additional information see [here](https://github.com/demisto/demisto-sdk/blob/master/demisto_sdk/commands/validate/README.md#validate).
+
+### lint
+Use this command to make sure your `.py` and `.ps1` files are up to Cortex XSOAR standards.
+This command runs several libraries to validate your code these include:
+ - [Flake8](https://pypi.org/project/flake8/) - Making sure your code is up to PEP8 standards.
+ - [MyPy](https://pypi.org/project/mypy/) - Validating type annotations, assignments and additional python checks.
+ - [Vulture](https://pypi.org/project/vulture/) - Finds unused code.
+ - [Bandit](https://pypi.org/project/bandit/) - Finds security issues.
+ - XSOAR Linter - An internal linter used to identify XSOAR specific restrictions and provide best practice advice for your code.
+ - Pylint and Pytest - Used to run your attached unit test (in the `_test.py`) file on the integration/script's Docker.
+ - PowerShell test and analyze - Code linters for PowerShell.
+
+At the end of the command a short report will appear with all the error and warnings found, as well as failed and passed unit tests.
```

Comment:
Add links to: 

* https://xsoar.pan.dev/docs/integrations/linting
* https://xsoar.pan.dev/docs/integrations/unit-testing

Reply:
done